docs/local.md

```diff
@@ -35,7 +35,7 @@
 
  - [install **helm3**](https://helm.sh/docs/intro/install/) to deploy k8gb and related test workloads
 
- - [install **k3d**](https://k3d.io/#installation) to run local [k3s](https://k3s.io/) clusters
+ - [install **k3d**](https://k3d.io/#installation) to run local [k3s](https://k3s.io/) clusters (minimum 4.0 version is required)
```

Comment:
I realised `4.0.0` didn't work properly, so we made upstream changes. 
Can you change to `4.2.0` ?

Reply:
Thanks, I recall that. udp loadbalancer thingie. Sure, nice catch!